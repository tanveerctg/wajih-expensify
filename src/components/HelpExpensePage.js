import React from 'react';

const HelpExpensePage=()=>(
  <div>
    This is the HelpExpensePage
  </div>
);

export default HelpExpensePage;