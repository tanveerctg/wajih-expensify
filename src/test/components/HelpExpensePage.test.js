import toJson from 'enzyme-to-json';
import React from 'react';
import { shallow } from 'enzyme';
import HelpExpensePage from  '../../components/HelpExpensePage';
import setupTests from '../setupTests';


test("should render HelpExpensePage component correctly",()=>{
  const wrapper=shallow(<HelpExpensePage />);
  expect(toJson(wrapper)).toMatchSnapshot();
})