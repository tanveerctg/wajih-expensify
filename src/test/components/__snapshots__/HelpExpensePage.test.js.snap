// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HelpExpensePage component correctly 1`] = `
<div>
  This is the HelpExpensePage
</div>
`;
